src/base.config.ts

```diff
@@ -190,7 +209,7 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 						getUMDCompatLoader({ bundles: args.bundles }),
 						{
 							loader: 'ts-loader',
-							options: { onlyCompileBundledFiles: true, instance: jsonpIdent, transpileOnly: true, compilerOptions }
+							options: tsLoaderOptions
```

Comment:
Need to remove `onlyCompileBundledFiles` and `transpileOnly` to make sure that Typescript gathers all the types needed by the custom element transformer.